### PR TITLE
Dbschema

### PR DESCRIPTION
**Pull request by nicosesma**
Set up dev environment and setup the schema for database with seeded data
@jrob8577

### CODE REVIEW

**Comment by nicosesma** [comment]
Did we set up babel correctly? @jrob8577

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 Create a simple content management system that allows users to add, delete, or update books.  Books entered in the system can be viewed in a listing, or searched for using basic searches (title, author, genre).
 
+## Instructions:
+ * Clone the project from github
+ * NPM Install
+ * Create lgbookstore database in Postgres SQL
```
Comment (jrobcodes) [inline README.md]: How do you do this step?

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+('Young Adult'),
+('Historical Drama'),
+('Romance'),
+('Horror'),
+('Satire');
+
+--- Join Tables Start Here
+
+INSERT INTO book_authors
+SELECT
```
Comment (jrobcodes) [inline schema.sql]: For clarity, I would wrap all of the embedded `SELECT`s in parentheses.

Comment (nicosesma) [reply schema.sql]:
``` sql
INSERT INTO book_genres
  ( SELECT books.id, genres.id FROM  books 
    CROSS JOIN  genres 
    WHERE   books.title = 'The Lord of the Rings' AND  genres.name = 'Fantasy'
  );
```

**Pull request merged**